File: TransCoder_tokenized_test_set_functions/java/MINIMUM_ROOMS_FOR_M_EVENTS_OF_N_BATCHES_WITH_GIVEN_SCHEDULE.java
static int findMinRooms ( String slots [ ], int n, int m ) {
  int counts [ ] = new int [ m ];
  for ( int i = 0;
  i < m;
  i ++ ) counts [ i ] = 0;
  for ( int i = 0;
  i < n;
  i ++ ) for ( int j = 0;
  j < m;
  j ++ ) if ( slots [ i ] . charAt ( j ) == '1' ) counts [ j ] ++;
  int max = - 1;
  for ( int i = 0;
  i < m;
  i ++ ) if ( max < counts [ i ] ) max = counts [ i ];
  return max;
}

File: TransCoder_tokenized_test_set_functions/python/MINIMUM_ROOMS_FOR_M_EVENTS_OF_N_BATCHES_WITH_GIVEN_SCHEDULE.py
def findMinRooms ( slots, n, m ) :
    counts = [ 0 ] * m;
    
    for i in range ( n ) :
        for j in range ( m ) :
            if ( slots [ i ] [ j ] == '1' ) :
                counts [ j ] += 1;
                
    return max ( counts );
